src/CodeGenerator/src/Generator/InputGenerator.php

```diff
@@ -561,7 +561,7 @@ private function stringify(string $variable, Member $member, string $part): stri
         $shape = $member->getShape();
         switch ($shape->getType()) {
             case 'timestamp':
-                $format = strtoupper($shape->get('timestampFormat') ?? ('header' === $part ? 'rfc822' : 'iso8601'));
+                $format = strtoupper($shape->get('timestampFormat') ?? ('header' === $part ? 'rfc822' : 'atom'));
```

Comment:
`shape->get('timestampFormat')` might return `iso8601`.

```suggestion
                $format = strtoupper($shape->get('timestampFormat') ?? ('header' === $part ? 'rfc822' : 'atom'));
                if ($format === 'ISO8601') {
                    $format = 'ATOM'
                }
```

Reply:
You are right, thank you